pkg/proxy/ipvs/proxier.go

```diff
@@ -2004,6 +2004,12 @@ func (proxier *Proxier) syncEndpoint(svcPortName proxy.ServicePortName, onlyNode
 				newEndpoints = localReadyTerminatingEndpoints
 			}
 		}
+
+		// Insert non-local endpoints to mimic the behavior of the iptables proxier
```

Comment:
It would be helpful to add more comments on the justifcation for this change. Specifically about how this only applies for nodes that don't have local endpoints anyways.